### ISSUE TITLE
can not read data 

### ISSUE THREAD

**Issue by ymmooot** [ymmooot/nuxt-jsonld#347]
```javascript

// Mixin
export default {
    head() {

        const ACTIVE_PAGINATION = this.pagination.page + 1;

    },
};

```

> Cannot read property 'page' of undefined.

This is a mixin in a page, and `this.pagination.page` is coming from the asyncData method 🤔

On `v1.4.5` still no errors

_Originally posted by @LuXDAmore in https://github.com/ymmooot/nuxt-jsonld/issues/342#issuecomment-617753858_

**Comment by ymmooot** [comment]
Hi @LuXDAmore @AndrewBogdanovTSS .
I have just published v1.4.9. Could you try the new version?

I hope it will solve the problem 🙏

**Comment by LuXDAmore** [comment]
@ymmooot Everything seems fine! 🚀 

Thank you very much  💯
